### DISCUSSION TITLE
Optiga Trust M OPN

### DISCUSSION THREAD

**Discussion by SoftAvocado** [Infineon/optiga-trust-m#113] [Q&A] [answered]
Hello. I need to get an OPN  (Orderable Part Number) of the chip. Infenion FAQ says i can only get it from Barcode Packing Label and other order documents. But is there any way I can read an OPN from the chip itself?

**Comment by koblibri** [comment, marked as answer]
Hi,

the OPN depends a bit on your chosen configuration.
For OPTIGA Trust M v3 it is either SLS32AIA010MKUSON10XTMA2 (for the STR Temp-Range) or SLS32AIA010MLUSON10XTMA2 (for the ETR Temp-Range). You can directly order and see the OPNs on our Webpage: https://www.infineon.com/optiga-trust-m

For Trust M Express (SLS32AIA010MLUSON10XTMA9), see here: https://infineon.com/optiga-trust-m-express

For everything else, please get in touch with your local (distribution) sales-person, as we cannot handle these things in Github.

Lastly, reading the OPN from the chip is not possible. The OPN is a Sales-Code, while the chip only holds information on things like  build-date and -number.
